rcsbapi/data/schema.py

```diff
@@ -56,6 +58,8 @@ def __init__(self, pdb_url):
         self.node_index_dict = {}
         self.edge_index_dict = {}
         self.type_fields_dict = {}
+        self.name_description_dict = {}
+        self.seen_names = set()
```

Comment:
I think you need to initialize by running `extract_name_description` here because when I initialize, this dict is showing as empty. (You can put this at line 74-78 where a bunch of initialization functions are run)

Reply:
move line 67 to 73?

Reply:
no, currently, extract_name_description isn't being run in __init__ so `name_description_dict` shows as empty

Reply:
I just mean line 74 is where several similar functions are being run, like recurse_build_schema

Reply:
i am running it on line 67, do you have the latest changes?

Reply:
oh sorry, I pulled again and I see it now